src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -56,3 +57,9 @@ def get(self, svc_id):
             'osd_metadata': mgr.get_metadata('osd', svc_id),
             'histogram': histogram,
         }
+
+    @RESTController.Resource('POST')
+    def scrub(self, svc_id, deep=False):
+        api_scrub = "osd deep-scrub" if deep else "osd scrub"
+        CephService.send_command("mon", api_scrub, who=svc_id)
+        return svc_id
```

Comment:
a single integer is not a valid json response. Why do you need to return the osd id? the frontend know which id used to make the request.